templates/repo/issue/new_form.tmpl

```diff
@@ -15,7 +15,7 @@
 					<div class="field">
 						<input name="title" id="issue_title" placeholder="{{.i18n.Tr "repo.milestones.title"}}" value="{{.title}}" tabindex="3" autofocus required maxlength="255">
 						{{if .PageIsComparePull}}
-							<div class="title_wip_desc">{{.i18n.Tr "repo.pulls.title_wip_desc" (index .PullRequestWorkInProgressPrefixes 0| Escape) | Safe}}</div>
+							<div class="title_wip_desc"{{if .PageIsComparePull}} data-wip-prefixes="{{Json .PullRequestWorkInProgressPrefixes}}"{{end}}>{{.i18n.Tr "repo.pulls.title_wip_desc" (index .PullRequestWorkInProgressPrefixes 0| Escape) | Safe}}</div>
```

Comment:
The if here seems superfluous

Reply:
Probably is, I just preserved it as it was present in the inline script below. You think it's fine to remove still?

Reply:
Removed it.

Reply:
Yeah it's already inside an {{if .PageIsComparePull}} one line above

Reply:
Ah, missed that.